From 6cf8099cfa87b43de4f2103ece7e60144147a225 Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <Michal@Hrusecky.net>
Date: Mon, 19 Feb 2018 21:26:48 +0100
Subject: [PATCH] scripts: Don't skip kernel and libc in Packages generation

It kinda breaks dependencies as it looks like libc and kernel are missing from
repositories.

Signed-off-by: Michal Hrusecky <Michal@Hrusecky.net>
---
 scripts/ipkg-make-index.sh | 2 --
 1 file changed, 2 deletions(-)

diff --git a/scripts/ipkg-make-index.sh b/scripts/ipkg-make-index.sh
index f6f15d7dd8..910e9872c3 100755
--- a/scripts/ipkg-make-index.sh
+++ b/scripts/ipkg-make-index.sh
@@ -14,8 +14,6 @@ for pkg in `find $pkg_dir -name '*.ipk' | sort`; do
 	empty=
 	name="${pkg##*/}"
 	name="${name%%_*}"
-	[[ "$name" = "kernel" ]] && continue
-	[[ "$name" = "libc" ]] && continue
 	echo "Generating index for package $pkg" >&2
 	file_size=$(stat -L -c%s $pkg)
 	sha256sum=$(mkhash sha256 $pkg)
-- 
2.20.1

